conda/recipes/conda_build_config.yaml

```diff
@@ -26,9 +26,6 @@ libboost_devel:
 libboost_python_devel:
   - 1.84 # Aim to follow conda-forge
 
-cmake:
-  - '>=3.21.0'
```

Comment:
I think it's safer to keep this pin in place since it protects against older versions that won't work with our current cmake setup.

Reply:
Does that mean this PR should be closed?

Reply:
It can be closed if we rather keep cmake pinned. Maybe we need a comment in the dependency failure spreadsheet that we decided not to unpin it.